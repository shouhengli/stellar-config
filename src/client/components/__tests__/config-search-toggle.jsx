const React = require('react');
const renderer = require('react-test-renderer');
const Toggle = require('../config-search-toggle.jsx');

describe('component config-search-toggle', () => {
  test('is clickable', () => {
    const props = {
      handleClick: jest.fn()
    };

    const tree = renderer.create(<Toggle {...props} />).toJSON();
    tree.props.onClick();

    expect(tree).toMatchSnapshot();
    expect(props.handleClick).toHaveBeenCalledTimes(1);
  });
});
