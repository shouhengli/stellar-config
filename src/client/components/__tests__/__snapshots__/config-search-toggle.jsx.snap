// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component config-search-toggle is clickable 1`] = `
<span
  className="button"
  onClick={[Function]}
>
  <span
    className="icon"
  >
    <i
      className="fa fa-search"
    />
  </span>
</span>
`;
